components/net/http_cache.rs

```diff
@@ -75,7 +74,8 @@ struct MeasurableCachedResource {
     raw_status: Option<(u16, Vec<u8>)>,
     url_list: Vec<ServoUrl>,
     expires: Duration,
-    last_validated: Tm,
+    #[ignore_malloc_size_of = "Platform-specific Instant object isn't measured"]
```

Comment:
Do we need the explicit ignore_malloc_size_of here? It seems like [there is an implementation for the Instant type](https://github.com/servo/stylo/blob/65f8d1316a0966401bcfb9fa7dd5e3659a1605b2/malloc_size_of/lib.rs#L957).

Reply:
Nope. I've removed this.